source/_docs/configuration.markdown

```diff
@@ -35,19 +35,6 @@ _If you use {% term "Home Assistant Operating System" %}, you can find `configur
 
 _If you use {% term "Home Assistant Core" %} , you can find `configuration.yaml` in the config folder passed to the `hass` command (default is `~/.homeassistant`)._
 
-## Reloading configuration changes
-
-Most integrations in Home Assistant that do not interact with {% term devices %} or {% term services %} can reload changes made to their configuration in `configuration.yaml`.
-
-1. To reload configuration changes, go to {% my server_controls title="**Developer Tools** > **YAML**" %} and scroll down to the YAML configuration reloading section (alternatively, hit "c" anywhere in the UI and search for "reload").
-   - You are presented with a list of integrations, such as **Automations** or **Conversation**.
-
-    ![Reload configuration changes](/images/docs/configuration/reloading_config.png)
-
-2. If the integration is listed, select it to reload the settings.
-3. If integration is not listed, you need to restart Home Assistant for changes to take effect:
-   - [Validate the configuration](#validating-the-configuration). Then, select the **Restart** button.
-
 ## Validating the configuration
 
 After changing configuration or automation files, check if the configuration is valid.
```

Comment:
This part is actually no longer valid and/or needed. Reload/restart of Home Assistant will already run validation.

Reply:
I see. 
we refer to this procedure from this page https://www.home-assistant.io/docs/configuration/yaml/
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/03e0f3b4-5c1a-4733-9bc3-0b94bf5c7e45)

this section ould probably be removed as well?

Reply:
Yeah I guess that makes sense. Back in the day, you had to check before restarting (to ensure HA would start). This safeguard is nowadays just part of the restart/reload itself (and above all, Home Assistant will **always** start now).

Reply:
But what if you just want to validate without immediately restarting? I would not delete this section, but rewrite it with text that validation always happens during a restart.

Reply:
another question: maybe we have a bit of a Matryoshka doll situation here.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/256ff0eb-4085-41df-835d-4d97fb5ff6de)
The Validation section points to the configuration check procedures for the different installation types.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/a845830c-b0dd-48b7-89dd-46f95d97babd)
for Container
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/56ea0196-f51c-4341-9926-d3f9d25ecca5)

Are they all no longer needed?

Reply:
It is not needed for before restarting, but its needed for a validation check when you are creating things.